stablehlo/reference/Ops.h

```diff
@@ -23,25 +23,26 @@ limitations under the License.
 namespace mlir {
 namespace stablehlo {
 
-Tensor eval(AddOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(AndOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(CeilOp op, const Tensor &operand);
-Tensor eval(ConstantOp op);
-Tensor eval(CosineOp op, const Tensor &operand);
-Tensor eval(FloorOp op, const Tensor &operand);
-Tensor eval(IotaOp op);
-Tensor eval(MaxOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(MinOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(MulOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(NegOp op, const Tensor &operand);
-Tensor eval(NotOp op, const Tensor &operand);
-Tensor eval(OrOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(ReshapeOp op, const Tensor &operand);
-Tensor eval(SineOp op, const Tensor &operand);
-Tensor eval(SubtractOp op, const Tensor &lhs, const Tensor &rhs);
-Tensor eval(TanhOp op, const Tensor &operand);
-Tensor eval(TransposeOp op, const Tensor &operand);
-Tensor eval(XorOp op, const Tensor &lhs, const Tensor &rhs);
+Tensor eval_add(Type resultType, const Tensor &lhs, const Tensor &rhs);
+Tensor eval_and(Type resultType, const Tensor &lhs, const Tensor &rhs);
+Tensor eval_ceil(Type resultType, const Tensor &operand);
+Tensor eval_constant(const ElementsAttr &value);
+Tensor eval_cosine(Type resultType, const Tensor &operand);
+Tensor eval_floor(Type resultType, const Tensor &operand);
+Tensor eval_iota(Type resultType, uint64_t iotaDimension);
```

Comment:
In TypeInference.h, we're using signed integers in these situations, which would be `int64_t` in this case.

Reply:
Thanks for catching this!